requirements.txt

```diff
@@ -0,0 +1,4 @@
+numpy
+pandas
+matplotlib
+sklearn
```

Comment:
Create one file `CONTRIBUTION.md` and write the setup procedure for it

Reply:
okay